client/utils/withRouteProps.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
the name of this file is camel case and is not consistent with the rest of the codebase, please fix

Reply:
Renamed :+1: